lighthouse-core/gather/connections/connection.js

```diff
@@ -92,7 +92,7 @@ class Connection {
     const object = JSON.parse(message);
     // Remote debugging protocol is JSON RPC 2.0 compiant. In terms of that transport,
     // responses to the commands carry "id" property, while notifications do not.
-    if (object.id) {
+    if (this._callbacks.has(object.id)) {
```

Comment:
what do you think about making this more explicit than treating it like an event if no callback (it's not really an event since the message does have an `id`)? Could do a third case with a call to `formatProtocol` with an explicit `?????` (and no emit).

Also with level `error` so it hits `log.error`? Not sure about that one if this is a regular occurrence in devtools land